src/acquisition/covid_hosp/common/database.py

```diff
@@ -45,7 +45,7 @@ def __init__(self,
     self.connection = connection
     self.table_name = table_name
     self.hhs_dataset_id = hhs_dataset_id
-    self.publication_col_name = "issue" if table_name == 'covid_hosp_state_timeseries' else \
+    self.publication_col_name = "issue" if table_name == 'covid_hosp_state_timeseries' or table_name == "covid_hosp_state_daily" else \
```

Comment:
this line is getting a little gnarly, but we will fix it with the other yaml-schema changes so you can leave it for now